phpBB/phpbb/db/migration/data/v330/extensions_composer_2.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v330;
+
+class extensions_composer_2 extends \phpbb\db\migration\migration
+{
+	public function update_data()
+	{
+		$repositories = json_decode($this->config, true);
```

Comment:
Shouldn't this be `json_decode($this->config->get(''exts_composer_repositories''), true)`?

Reply:
indeed...

Reply:
fixed